FirebaseStorage/Sources/Public/FirebaseStorage/FIRStorageReference.h

```diff
@@ -35,25 +35,25 @@ NS_ASSUME_NONNULL_BEGIN
 @interface FIRIMPLStorageReference : NSObject
```

Comment:
This is no longer a public header and will cause a messy conflict with #9598. Please revert

Reply:
Done.